ee/clickhouse/queries/experiments/test_experiment_result.py

```diff
@@ -1,24 +1,138 @@
 import unittest
+from typing import List
 
-from ee.clickhouse.queries.experiments.funnel_experiment_result import (
-    ClickhouseFunnelExperimentResult,
-    Variant,
-    probability_B_beats_A,
-)
+from numpy.random import default_rng
+
+from ee.clickhouse.queries.experiments.funnel_experiment_result import ClickhouseFunnelExperimentResult, Variant
 from ee.clickhouse.queries.experiments.trend_experiment_result import ClickhouseTrendExperimentResult
 from ee.clickhouse.queries.experiments.trend_experiment_result import Variant as CountVariant
 from ee.clickhouse.queries.experiments.trend_experiment_result import probability_B_beats_A_count_data
 
 
+def simulate_winning_variant_for_conversion(target_variant: Variant, variants: List[Variant]) -> float:
```

Comment:
I didn't find a good way to test my calculations, except trying to calculate them 2 different ways, and ensuring the results are same for both ways.